api_sentry.py

```diff
@@ -0,0 +1,90 @@
+import os
+import sys
+
+import pandas as pd
+# import numpy as np
+# import json
+
+
+from lib.sentry_conn import APIClient
+
+# TODO: Database
+# from database import (
+#    Database,
+#    ReportSentryIssues
+# )
+
+
+class Sentry:
+
+    def __init__(self):
+        try:
+            self.client = APIClient(os.environ['SENTRY_HOST'])
+            self.client.api_token = os.environ['SENTRY_API_TOKEN']
+            self.client.organization_slug = \
+                os.environ['SENTRY_ORGANIZATION_SLUG']
+            self.project_id = os.environ['SENTRY_PROJECT_ID']
+            self.project_slug = 'mozilla'
+        except KeyError:
+            print("ERROR: Missing testrail env var")
+            sys.exit(1)
+
+    # API: Issues
+    # Question: Should we just the for_review issues?
+    # TODO: Query everyday. Only the unassigned issues past day.
```

Comment:
question.. are all issues in sentry crashes? we may need to sync with devs to know which ones the monitor, if there is a special flag or something that we should know to filter those in the request or later

Reply:
I don't know a special flag/sign/signature at the moment for the crashes. 

`for_review` is a good place to begin: This flag denotes issues that don't have an owner.